docs/practical-guides/firebase-config/index.md

```
@@ -0,0 +1,140 @@
+---
+sidebar_position: 1
+---
+
+# Configuración de Firebase
+
+### Configurando Firebase para varios ambientes
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)